cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/infrastructure/persistence/aws/cloudsearch/AmazonCloudSearchDomainClientBuilder.java

```diff
@@ -23,7 +23,8 @@
 public class AmazonCloudSearchDomainClientBuilder {
 
     public static AmazonCloudSearchDomain build(final AWSCredentials awsCredentials, final String endpoint) {
```

Comment:
Would be good to have a new public method which only takes the endpoint, which simply passed null into the existing factory method?